### ISSUE TITLE
Groups must not have counter for subscriptions, likes and comments in the header

### ISSUE THREAD

**Issue by iAdramelk** [pepyatka/pepyatka-html#265]
Right now groups page header have links and counters for subscribers, likes and comments (see screenshot). 

![screen shot 2015-04-27 at 02 34 40](https://cloud.githubusercontent.com/assets/451207/7339384/007c1ea2-ec86-11e4-8830-48c705c94af0.png)

But spec says that there is no such a thing for groups: https://drive.google.com/open?id=1q0-fHGqhmtRihut0X1o1iQYIThFN46pouBoo3-YJgfg&authuser=0

**Comment by smallq-git** [comment]
What group info looked like in Frf:

![image](https://cloud.githubusercontent.com/assets/7983303/7339407/1df4fc66-ec76-11e4-809b-983f62070a2f.png)

**Comment by epicmonkey** [comment]
I'd rephrase this: we should not show any counters that == 0.

**Comment by iAdramelk** [comment]
@epicmonkey From that I understand, group can't physically have subscriptions or leave comments. So there is no reason to have it at all.

**Comment by smallq-git** [comment]
@epicmonkey zero counters should be displayed - this is correct behavior.
But  a group is not a user, they don't like, comment or subscribe to anyone.

**Comment by smallq-git** [comment]
Fixed:
![image](https://cloud.githubusercontent.com/assets/7983303/8046040/70b533da-0e40-11e5-83a0-5a3c6a43deba.png)
